labgrid/remote/config.py

```diff
@@ -23,7 +23,8 @@ def __attrs_post_init__(self):
             raise NoConfigFoundError(
                 "{} could not be found".format(self.filename)
             )
-        rendered = template.render()
+        import os
```

Comment:
Please move this import to the top of the file.

Reply:
> The full resource name already includes the exporter's hostname. Wouldn't that be enough for this use-case? How does `labgrid-client resources` look in your case?

Yes, you are of-course right.  No point in having the same string appear twice in the resource name.

I still believe the change is a worthy improvement.